test/e2e/common/utils.py

```diff
@@ -46,3 +46,24 @@ def predict(service_name, input_json):
         logging.info("Got response code %s, content %s", response.status_code, response.content)
         probs = json.loads(response.content.decode('utf-8'))["predictions"]
         return probs
+
+def explain(service_name, input_json):
+    isvc = KFServing.get(service_name, namespace=KFSERVING_TEST_NAMESPACE)
+    # temporary sleep until this is fixed https://github.com/kubeflow/kfserving/issues/604
+    time.sleep(10)
+    api_instance = client.CoreV1Api(client.ApiClient())
+    service = api_instance.read_namespaced_service("istio-ingressgateway", "istio-system", exact='true')
+    if service.status.load_balancer.ingress is None:
```

Comment:
Can we generalize this to a helper to sahre between predict and explain?

Reply:
Sound good. Will do.